src/Workspaces/Core/Portable/Shared/Extensions/LineSpan.cs

```diff
@@ -19,7 +19,7 @@ internal struct LineSpan : IEquatable<LineSpan>
         public int End { get; private set; }
 
         public static LineSpan FromBounds(int start, int end)
-            => new LineSpan
+            => new()
             {
                 Start = start,
                 End = end
```

Comment:
@sharwell @jcouv format analyzer is complaining about these lines now.  Is this a known issue?  It looks like it wants to dedent the body by 4.

Reply:
This is fixed by #47442

Reply:
Gotcha.  thoughts on what to do here?  should we wait for that fix to flow out?  or should i revert all the changes here that trigger that?

Reply:
> should we wait for that fix to flow out

Unfortunately this is no longer realistic due to a recent design change in the code style layer. We tied the formatter analyzer to the compiler release, so this will not be usable until 16.8 Preview 4 is publicly available _and_ the build is updated to use it.

> thoughts on what to do here?

Either of the following are fine with me:

* Avoid making changes that trigger this
* Change the formatting to the sub-optimal form here, and it will be corrected automatically by whatever future pull request updates the formatting analyzer version

Reply:
> Avoid making changes that trigger this

I have gone with this.